pages/dmp_supporting_info/support_02_about_research.md

```diff
@@ -41,7 +41,7 @@ A project description/abstract can be provided in two ways: By linking to existi
 ### Did you or do you plan to apply for external project funding?
 You should list all funding that enables the project or that you applied for.
 
-Add funder by clicking the +Add button. Start typing the funder name (in English or Norwegian) in the text box to see a list of suggestions based on the Research Organization Registry (ROR). Funder names should be added from the Research Organization Registry (ROR) when possible. In case the funder is not found in the Research Organization Registry (ROR), type the full name and click outside the text box, and the string will be saved.
+Add funder by clicking the +Add button. Start typing the funder name (in English or Norwegian) in the text box to see a list of suggestions based on the Crossref funder registry. In case the funder is not found in the Crossref funder registry, type the full name and click outside the text box, and the string will be saved.
```

Comment:
Comment not to fix: Crossref Funder ID is about to transition to ROR. Eventually the Funder ID will then be a part of ROR
https://www.crossref.org/blog/open-funder-registry-to-transition-into-research-organization-registry-ror/
(so we have to update once this is done).